programs/client/Client.cpp

```diff
@@ -1264,6 +1264,15 @@ void Client::processConfig()
     global_context->setQueryKindInitial();
     global_context->setQuotaClientKey(config().getString("quota_key", ""));
     global_context->setQueryKind(query_kind);
+
+    if (is_multiquery && !global_context->getSettingsRef().input_format_values_allow_data_after_semicolon.changed)
+    {
+        Settings settings = global_context->getSettings();
+        settings.input_format_values_allow_data_after_semicolon = true;
+        /// Do not send it to the server
+        settings.input_format_values_allow_data_after_semicolon.changed = false;
```

Comment:
That's too tricky, but maybe it's ok

Reply:
The same is used for distributed queries, so I think it is OK:

```
src/Interpreters/ClusterProxy/executeQuery.cpp:        new_settings.max_concurrent_queries_for_user.changed = false;
src/Interpreters/ClusterProxy/executeQuery.cpp:        new_settings.max_memory_usage_for_user.changed = false;
src/Interpreters/ClusterProxy/executeQuery.cpp:            new_settings.force_optimize_skip_unused_shards.changed = false;
src/Interpreters/ClusterProxy/executeQuery.cpp:            new_settings.optimize_skip_unused_shards.changed = false;
src/Interpreters/ClusterProxy/executeQuery.cpp:        new_settings.offset.changed = false;
src/Interpreters/ClusterProxy/executeQuery.cpp:        new_settings.limit.changed = false;
```